integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -18,6 +18,13 @@ if [ "$ID" != ubuntu  ]; then
 	exit
 fi
 
+# Currently, Kubernetes tests are not working with initrd.
+if [ "$TEST_INITRD" = yes ] && [ "$AGENT_INIT" = yes ] && [ "$OSBUILDER_DISTRO" = alpine ]; then
```

Comment:
- Where is `OSBUILDER_DISTRO` set?
- Yep, I had to be careful with initrd's for https://github.com/kata-containers/osbuilder/pull/102, see:
  https://github.com/kata-containers/osbuilder/blob/master/tests/image_creation.bats#L281. But could this test just use the first to tests?:
  ```
  if [ "$TEST_INITRD" = yes ] && [ "$AGENT_INIT" = yes ]; then
  ```